src/main/java/org/zalando/nakadi/domain/BatchItem.java

```diff
@@ -1,13 +1,15 @@
 package org.zalando.nakadi.domain;
 
-import java.util.Optional;
-import javax.annotation.Nullable;
 import org.json.JSONObject;
 
+import javax.annotation.Nullable;
+import java.util.Optional;
+
 public class BatchItem {
     private final BatchItemResponse response;
     private final JSONObject event;
     private String partition;
+    private int brokerId;
```

Comment:
Batch item is a view object, and with creation of brokerId in this object we are linking to kafka implementation.